R/json.R

```diff
@@ -1,8 +1,8 @@
 #' @importFrom jsonlite validate fromJSON toJSON
 #' @noRd
-safeFromJSON <- function(txt) {
+safeFromJSON <- function(txt, ...) {
   if (!validate(txt)) {
     stop("Argument 'txt' is not a valid JSON string.")
   }
-  fromJSON(txt)
+  fromJSON(txt, ...)
```

Comment:
Switch to `jsonlite::parse_json` here and default the parser to `simplifyVector = TRUE`

Reply:
still relevant?

Reply:
Yes

`jsonlite::parse_json` assumes the input is a string to be parsed, not a string or file.  However, `jsonlite::parse_json` changes the default behavior of `simplifyVectors` to `FALSE` when compared `jsonlite::fromJSON`. It should be set to `TRUE` for backwards compatibility.

Reply:
Done in #587